shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/screens/AboutApp.kt

```diff
@@ -0,0 +1,21 @@
+package org.jetbrains.kotlinconf.screens
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import kotlinconfapp.shared.generated.resources.Res
+import kotlinconfapp.shared.generated.resources.arrow_left_24
+import org.jetbrains.compose.resources.painterResource
+import org.jetbrains.kotlinconf.LocalNavController
+import org.jetbrains.kotlinconf.ui.components.StyledText
+
+@Composable
+fun AboutApp() {
+    val navController = LocalNavController.current
+    Column {
+        Image(painterResource(Res.drawable.arrow_left_24), "back", modifier = Modifier.clickable { navController.popBackStack() })
```

Comment:
What's your take on using `popBackStack` vs `navigateUp` in general?

This video reasons for using `navigateUp` instead of `popBackStack`: https://www.youtube.com/watch?v=y2zLFONuk7c

Reply:
Thanks for the link, haven't thought about it before in that way.
As many commenters have pointed out, the issue is not with `popBackStack`/`navigateUp`, but rather with preventing duplicate clicks or, in general, any user actions during the navigation transition.
As our own Ivan Matkov suggested in the comments there, we may check Lifecycle state to prevent this.

On a side note, the Back button behaves differently on iOS, as there you're always navigating inside the app, while on Android with `navigateUp` you may get back to a previous app, depending on how our app was launched.

Reply:
Ah yes, I've also heard the solution with checking the lifecycle. I think the core issue is that people want an API that's a single function and safe to use by default.

I guess they can all write their own `popBackStackSafely()` functions, which are `@Composable` so that they can reach out to the lifecycle to check it before doing a pop.